apps/mobile/src/features/Discover/common/ConfirmHWConnectionModal.tsx

```diff
@@ -32,9 +32,9 @@ export const useConfirmHWConnectionModal = () => {
   const {openModal, closeModal} = useModal()
   const strings = useStrings()
   const confirmHWConnection = useCallback(
-    ({onConfirm, onClose, onCancel}: Props) => {
+    ({onConfirm, onClose: _onClose, onCancel}: Props) => {
```

Comment:
i think it can be deleted
```suggestion
    ({onConfirm, onCancel}: Props) => {
```

Reply:
I'm not so sure, I don't even know why it was deleted from the modals itself. Right now the state never changes to indicate that you have already seen the modal, because that used to be on the onClose. I didn't make the decision one way or the other because I just wanted to unlock the feature, but need to decide based on other places using it whether modals should get onClose again or not.